src/deluge/model/global_effectable/global_effectable.cpp

```diff
@@ -156,11 +168,23 @@ void GlobalEffectable::displayCompressorAndReverbSettings(bool on) {
 	}
 	else {
 		if (on) {
-			display->displayPopup(getCompressorModeDisplayName());
+			if (getSelectedClip()) {
+				// We are inside a clip (either audioclip or affect-entire kit): show Sidechain sync
+				if (compressor.syncLevel == SYNC_LEVEL_32ND) {
```

Comment:
Just tested and this doesn't work. It's changing between Slow and Fast with every press. I think you need to the same logic as in Sound::